### PR TITLE
setup a full webpage

### PR DESCRIPTION
**Pull request by MarkoKermi**
- [ ] Create a website consisting of 3 pages: Home, Calculator, and Quote.
- [ ] The general layout.
- [ ] Add my own styles to improve the look and feel.
- [ ] Use React basic styling (inline styles, importing CSS and pre-processed CSS files).

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable max-len */
+// import design from '../design/Home.css';
```
Comment (ichala) [inline src/components/Home.js]: - [ ] To keep your repo clean & professional , kindly remove any commented code from your code 🙏

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable max-len */
```
Comment (ichala) [inline src/components/Home.js]: - [ ] Linters helps you to deliver a clean & readable code , kindly remove all the disabling commands and try to fix them 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Created Routes ✔️ 
- App works as expected ✔️


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @MarkoKermi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights
Linters are passing ✔️
Created Routes ✔️
App works as expected ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**